x-pack/plugins/security_solution/public/detections/components/host_isolation/use_host_isolation_action.tsx

```diff
@@ -28,17 +36,30 @@ export const useHostIsolationAction = ({
   isHostIsolationPanelOpen,
   onAddIsolationStatusClick,
 }: UseHostIsolationActionProps): AlertTableContextMenuItem[] => {
+  const hasActionsAllPrivileges = useKibana().services.application.capabilities.save;
```

Comment:
ahhh. I had missed this before. Good - so looks like we are already checking that the user has `all` privilege to Actions and Connectors - right?